silk/node/wpantund_base.py

```diff
@@ -476,20 +478,22 @@ def timed_ping6(self, ipv6_target, num_pings, payload_size=8, interface=None):
     #   UDP functionality
     #################################
 
-    def send_udp_data(self, target: str, port: int, message: str, source: str = None, hop_limit: int = 64):
+    def send_udp_data(self, target: str, port: int, message: str, source: str = None,
+                      src_port: int = random.randint(11200, 11400), hop_limit: int = 64, timeout: float = 10):
         """Perform netcat command to send UDP message to ipv6_target via port.
 
         Args:
             target (str): target address.
             port (int): target port.
             message (str): message to send.
             source (str, optional): source address. Defaults to None.
+            src_port (int, optional): source port. Defaults to None.
```

Comment:
default to a random port between 11200 and 11400 ?

Reply:
Ack